tests/conftest.py

```diff
@@ -69,8 +69,10 @@ def jwt_partial_payload() -> dict:
     }
 
 
-@pytest.fixture(scope="session")
-def full_access_authz_payload(jwt_partial_payload) -> dict:  # pylint: disable=redefined-outer-name
+@pytest.fixture(scope="session", name="full_access_authz_payload")
+def full_access_authz(
+    jwt_partial_payload: dict,
```

Comment:
Why can't it be placed in one line? To be honest, I prefer one line of declaration instead of three 😉